src/firetower/incidents/permissions.py

```diff
@@ -18,3 +20,48 @@ class IsAuthenticated(permissions.BasePermission):
 
     def has_permission(self, request: Request, view: "APIView") -> bool:
         return request.user and request.user.is_authenticated
+
+
+class IncidentPermission(permissions.BasePermission):
+    """
+    Permission class for incident CRUD operations.
+
+    - READ: User must have visibility to the incident (respects is_visible_to_user)
+    - CREATE: Any authenticated user can create
+    - UPDATE: Captain, reporter, participants, or superuser
```

Comment:
This limits adding participants to just existing participants/superusers but it feels like we should want everyone to be able to update incidents (except private ones I guess since they'd have advanced access control requirements?)